pkg/adapter/handlerConfig.gen.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// THIS FILE IS AUTOMATICALLY GENERATED.
+
+package adapter
+
+//"istio.io/mixer/template/checknothing"
+//"istio.io/mixer/template/listentry"
+//"istio.io/mixer/template/logentry"
+//"istio.io/mixer/template/metric"
+//"istio.io/mixer/template/quota"
+//"istio.io/mixer/template/reportnothing"
+
+// HandlerConfig represents configuration specific to each of the declared operator's handler configs.
+//
+// HandlerConfig contains adapter-specific configuration as well template specific Type information for instances
```

Comment:
HandlerConfig contains adapter-specific configuration, as well as template-specific type information for each of the templates supported by the adapter. Mixer prepares HandlerConfig and provides it to the adapter during the configuration process. The type information it contains describes the shape of the instances the adapter may receive at runtime. Mixer only supplies type information related to templates the adapter explicitly supports.

Reply:
nice.. thanks